docs/team/drwala.md

```diff
@@ -1,53 +1,98 @@
 
 * **Team tasks**:
   * Assisted in planning and implementation of forking workflow, including the norms the group will agree to
   * Assisted members in using Intellij to manage and resolve merge conflicts
   * Set up system of reviewing PRs within the team to enable a faster PR response time
-  * Set up [user stories](https://github.com/AY2021S2-CS2103-T14-1/tp/issues?q=is%3Aissue+author%3Adrwala+as+a) and [tasks](https://github.com/AY2021S2-CS2103-T14-1/tp/issues?q=is%3Aissue+author%3Adrwala+implement) for the team in various sprints
+    * Team members would now PM 2 people to review the PRs instead of waiting for PRs to be reviewed, increasing
+      PR turnaround time
+  * Set up [user stories](https://github.com/AY2021S2-CS2103-T14-1/tp/issues?q=is%3Aissue+author%3Adrwala+as+a) and 
+    [tasks](https://github.com/AY2021S2-CS2103-T14-1/tp/issues?q=is%3Aissue+author%3Adrwala+implement) for the team 
+    in various sprints
   * Created the script and recorded the demo video for all releases
 
 * **Community**:
-  * PRs reviewed (with non-trivial review comments): [\#116](https://github.com/AY2021S2-CS2103-T14-1/tp/pull/116), [\#103](https://github.com/AY2021S2-CS2103-T14-1/tp/pull/103), [\#285](https://github.com/AY2021S2-CS2103-T14-1/tp/pull/285), [\#318](https://github.com/AY2021S2-CS2103-T14-1/tp/pull/318); with full list of all [PRs reviewed](https://github.com/AY2021S2-CS2103-T14-1/tp/pulls?q=is%3Apr+reviewed-by%3Adrwala)
-  * Contributed to forum discussions (examples: [1](https://github.com/nus-cs2103-AY2021S2/forum/issues/114), [2](https://github.com/nus-cs2103-AY2021S2/forum/issues/242), [3](https://github.com/nus-cs2103-AY2021S2/forum/issues/254), [4](https://github.com/nus-cs2103-AY2021S2/forum/issues/305))
+  * PRs reviewed (with non-trivial review comments): 
+    [\#116](https://github.com/AY2021S2-CS2103-T14-1/tp/pull/116), 
+    [\#103](https://github.com/AY2021S2-CS2103-T14-1/tp/pull/103), 
+    [\#285](https://github.com/AY2021S2-CS2103-T14-1/tp/pull/285), 
+    [\#318](https://github.com/AY2021S2-CS2103-T14-1/tp/pull/318); 
+    with full list of all [PRs reviewed](https://github.com/AY2021S2-CS2103-T14-1/tp/pulls?q=is%3Apr+reviewed-by%3Adrwala)
+  * Contributed to forum discussions (examples: 
+    [1](https://github.com/nus-cs2103-AY2021S2/forum/issues/114), 
+    [2](https://github.com/nus-cs2103-AY2021S2/forum/issues/242), 
+    [3](https://github.com/nus-cs2103-AY2021S2/forum/issues/254) and
+    [4 - bug in AB3 diagrams](https://github.com/nus-cs2103-AY2021S2/forum/issues/305))
   * Bug reports made for other teams were fully accepted during PE-D: [PE-D](https://github.com/DrWala/ped)
-  * Filed [bug reports](https://github.com/AY2021S2-CS2103-T14-1/tp/issues?q=is%3Aissue+author%3Adrwala+bug) for my own team
+  * Filed bug reports for my own team:
+    [\#293](https://github.com/AY2021S2-CS2103-T14-1/tp/pull/293),
```

Comment:
I think it would be better if the bugs linked had their descriptions updated

Reply:
Fixed